cpp/src/arrow/ipc/read_write_test.cc

```diff
@@ -1727,6 +1727,61 @@ TEST(TestIpcFileFormat, FooterMetaData) {
   ASSERT_TRUE(out_metadata->Equals(*metadata));
 }
 
+TEST_F(TestWriteRecordBatch, CompressionRatio) {
+  // ARROW-8823: Calculating the compression ratio
```

Comment:
Nit: rename this test and update the comment now that we don't compute a ratio anymore?

Reply:
Done.